Tests/test_cleaning.py

```diff
@@ -0,0 +1,72 @@
+"""Tests for the cleaning module"""
+from unittest.mock import patch
+import pytest
+import pandas as pd
+from life_expectancy import cleaning
+from .fixtures.mock_date import data_raw_tsv, data_expect, data_tabular, data_with_floats
+from . import OUTPUT_DIR
+
+@pytest.fixture
+def fixture_raw():
```

Comment:
```suggestion
@pytest.fixture(scope="module")
def fixture_raw() -> pd.DataFrame:
```

Without the scope, pytest will recreate this fixture every time a test calls it. On large code bases, this can make your test suite really slow.

(this comment also applies to the other fixtures)